ckan-postdev/docker-entrypoint.d/post_dev_tasks.sh

```diff
@@ -16,3 +16,9 @@ fi
 # echo "Loading test settings into ckan test-core.ini"
 # paster --plugin=ckan config-tool $SRC_DIR/ckan/test-core.ini \
 #     "ckan.harvest.mq.hostname = $TEST_CKAN_REDIS_HOSTNAME"
+
+paster --plugin=ckan config-tool $APP_DIR/production.ini "ckan.datagovuk.s3_aws_access_key_id = $CKAN_S3_AWS_ACCESS_KEY_ID"
```

Comment:
This section should probably be moved to the `start_ckan_development.sh` file rather than sit in the `post_dev_tasks.sh`